iaso/api/org_unit_change_request_configurations/views.py

```diff
@@ -88,13 +90,18 @@ def check_availability(self, request, *args, **kwargs):
             raise serializers.ValidationError("You must be logged in")
 
         project_id = ProjectIdSerializer(data=self.request.query_params).get_project_id(raise_exception=True)
+        oucrc_type = OrgUnitChangeRequestConfigurationTypeSerializer(data=self.request.query_params).get_type(
+            raise_exception=True
+        )
```

Comment:
I would probably have merged those in a single `Serializer` for this custom view.